scripts/run-artifactory.sh

```diff
@@ -1,17 +1,42 @@
 #!/usr/bin/env sh
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
+export ARTIFACTORY_VERSION=${ARTIFACTORY_VERSION:=7.27.10}
```

Comment:
`:=` ? never seen that - it's always been `${FOO:-7.27.10}`

Reply:
From https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

>${parameter:=word}
>If parameter is unset or null, the expansion of word is assigned to parameter. The value of parameter is then substituted. Positional parameters and special parameters may not be assigned to in this way.

`:-` makes more sense here. Will update.